Clinical-Genomics/scout
# Fetch refseq version from entrez and add it to clinvar form

This PR adds refseq versions to transcripts and makes them available to clinvar.
A much overdue fix #974
Could't find a solution to save version number when transcript database is populated via biomart but if you have a better solution I'll happily change this PR

**How to test**:
1. Now refseq versions are not available to clinvar, see pot1 variant example:
![image](https://user-images.githubusercontent.com/28093618/63949951-712be100-ca7b-11e9-923c-2701e3a59fc1.png)
1. Install a branch and try to submit to clinvar the same variant

**Expected outcome**:
refseq versions should be displayed in clinvar form:
![image](https://user-images.githubusercontent.com/28093618/63949830-43469c80-ca7b-11e9-9d97-00565ab104db.png)

The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
